features/submit_review_failure.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Create and submit reviews
```

Comment:
I think that both of these `.feature` files should be consolidated together. All of this functionality is related to the feature of submitting a review, so let's just put all of the scenarios for that into one file, maybe `submit_review.feature`?